test/e2e/nodepool_upgrade_test.go

```diff
@@ -110,6 +110,7 @@ func TestUpgradeNodePool(t *testing.T) {
 
 	e2eutil.EnsureNodeCountMatchesNodePoolReplicas(t, ctx, client, guestClient, hostedCluster.Namespace)
 	e2eutil.EnsureNoCrashingPods(t, ctx, client, hostedCluster)
+	e2eutil.NoticePreemptionOrFailedScheduling(t, ctx, client, hostedCluster)
```

Comment:
I think this needs rebase and so it can be added to the two existing tests in this file now.